Library/ViewModels/SurveyResponseViewModel.swift

```diff
@@ -142,3 +174,11 @@ private func isSurvey(request: URLRequest) -> Bool {
   guard case (.project(_, .survey, _))? = Navigation.match(request) else { return false }
   return true
 }
+
+private func requestInjectedWithFormData(with request: URLRequest, result: Any?) -> URLRequest {
+  var newRequest = request
+  if let formData = result as? String {
+    newRequest.httpBody = formData.data(using: .utf8)
```

Comment:
We cast the form data from `Any?` to `String` and set it as the raw `httpBody` of our request which we then prepare with our authentication headers.